From 2a100f11e8c5263bd49c3004b9794b282bbda465 Mon Sep 17 00:00:00 2001
From: fluffywaffles <skorlir@gmail.com>
Date: Mon, 17 Sep 2018 00:35:06 -0500
Subject: [PATCH] Custom: make TODOs red and italic

---
 colors/two-firewatch.vim | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/colors/two-firewatch.vim b/colors/two-firewatch.vim
index a5c61a4..c8725fd 100644
--- a/colors/two-firewatch.vim
+++ b/colors/two-firewatch.vim
@@ -380,7 +380,7 @@ if has('gui_running') || &t_Co == 88 || &t_Co == 256
   call <sid>X('Underlined',     s:duo_1,                '',          'underline')
   call <sid>X('Ignore',         '',                     '',          '')
   call <sid>X('Error',          s:syntax_color_removed, s:syntax_bg, 'bold')
-  call <sid>X('Todo',           s:syntax_color_added,   s:syntax_bg, '')
+  call <sid>X('Todo',           s:duo_2,                s:syntax_bg, 'italic')
   " }}}
 
   " Asciidoc highlighting ---------------------------------------------------{{{
-- 
2.19.0

